test/helper.js

```diff
@@ -1,3 +1,4 @@
+import './_cryptoPolyfill';
```

Comment:
Out of curiosity, what does ava complain about without the underscore?

Reply:
It was complaining that cryptoPolyfill didn't contain any test. According to [https://github.com/avajs/ava/blob/master/docs/05-command-line.md](ava's documentation) there are a bunch of ways to tell ava to ignore a file. "helpers" are ignored by default and so are file starting with _. Another way might be to specify the file in package.json but that sounded a bit overkilling to me.

Reply:
I see, thanks :+1: